api/api_test.go

```diff
@@ -84,3 +89,18 @@ func TestWriteJSONBody(t *testing.T) {
 		})
 	})
 }
+
+func TestAPISetup(t *testing.T) {
+	Convey("Excercise setUp", t, func() {
+		ctx := context.Background()
+		cfg, err := config.Get()
+		So(err, ShouldBeNil)
+		mockedDataStore := &storetest.StorerMock{}
+		permissions := mocks.NewAuthHandlerMock()
+
+		api := Setup(ctx, cfg, mux.NewRouter(), store.DataStore{Backend: mockedDataStore}, permissions)
+		So(api.navigationCacheMaxAge, ShouldResemble, 1800)
```

Comment:
Assertion fails as `api.navigationCacheMaxAge` set to string of value "1800.000000"

This is because the configuration value NavigationCacheMaxAge is converted into float64 before casting to string (see https://github.com/ONSdigital/dp-topic-api/pull/112/files#diff-bf98d5fab5bcfabded9769069e7318683f549ec91cff84c5f87a9ae3a3ca3d8dR51)